### ISSUE TITLE
Fix the macleod gamma aL2 to not be a constant.

### ISSUE THREAD

**Issue by reinhold-willcox** [TeamCOMPAS/COMPAS#1165]

**Comment by ilyamandel** [comment]
@reinhold-willcox -- see  OmegaAfterSynchronisation(), MassLossToFitInsideRocheLobe(), and Mass0ToMatchDesiredCoreMass() as examples for root solvers.

**Comment by ilyamandel** [comment]
@reinhold-willcox -- are you planning to work on this one?

**Comment by ilyamandel** [comment]
Agreed with @reinhold-willcox that this is not a priority right now; if anybody decides to work on this (or that they need it for a project), they can reopen.
